app/bundles/PointBundle/Config/config.php

```diff
@@ -63,6 +63,25 @@
                 'controller' => 'Mautic\PointBundle\Controller\Api\PointApiController::adjustPointsAction',
                 'method'     => 'POST',
             ],
+            'mautic_api_pointgroupsstandard' => [
+                'standard_entity' => true,
+                'name'            => 'pointGroups',
+                'path'            => '/points/groups',
+                'controller'      => \Mautic\PointBundle\Controller\Api\PointGroupsApiController::class,
```

Comment:
As per the new CS Fixer version:
```suggestion
                'controller'      => Mautic\PointBundle\Controller\Api\PointGroupsApiController::class,
```